devicons/devicon
# Convert all issue templates into issue forms

Things added/changed:

- Convert all issue templates into issue forms.
  - Also disables blank issues and provides an `Other` issue template.

## How to test

- This would have to be merged to `master` to test it directly. Although, check [**this**](https://github.com/EddieHubCommunity/support) repository and try to submit an issue.
